content/en/service_management/case_management/create_notifications_and_third_party_tickets.md

```diff
@@ -44,49 +44,52 @@ To get notified when a new case is created, create a view:
 
 
 
-## Third party tickets 
-In Project Settings, you can manage membership, configure the auto-closing of cases, and set up third-party integrations like Jira and ServiceNow. 
+## Third party tickets
+In Project Settings, you can manage membership, configure the auto-closing of cases, and set up third-party integrations like Jira and ServiceNow.
 
 ### Jira
 
 {{< img src="/service_management/case_management/settings/settings_jira.png" alt="Jira configuration options for case management settings" style="width:100%;" >}}
 
-1. Ensure the Jira integration is configured. 
-1. In Case Management project settings, enable **Jira** for manual Jira issue creation from the project. 
-1. Select a Jira account, a project to create issues in, and the desired issue type (such as story, epic, bug, or task). 
-1. You can opt into the automatic creation of a Jira issue for each case created in the project. 
+1. Ensure the Jira integration is configured.
+1. In Case Management project settings, enable **Jira** for manual Jira issue creation from the project.
+1. Select a Jira account, a project to create issues in, and the desired issue type (such as story, epic, bug, or task).
+1. You can opt into the automatic creation of a Jira issue for each case created in the project.
 1. For the following attributes—case title, description, assignee, comments, status, and priority—select one of the options below:
-  | Option     | Description    | 
-  | ---  | ----------- | 
+  | Option     | Description    |
+  | ---  | ----------- |
   |Once to Jira at case creation|The field syncs from Case Management to Jira only at the time the case is created. Subsequent changes are not reflected on either side.|
   |Two-way sync (bi-directional)|Changes in Case Management are reflected in Jira, and vice versa|
   |Don't sync|The field does not sync to Jira.|
-1. For case status and priority, select which values they map to on the Jira side. 
+1. For case status and priority, select which values they map to on the Jira side.
 1. Save changes.
 
-**Note**: A case can only be synced with one external resource at a time, per project. To enable Jira syncing, ServiceNow automatic creation and syncing must be disabled.
+Ongoing two-way syncing requires [webhook support](https://docs.datadoghq.com/integrations/jira/#configure-a-jira-webhook).
+
+**Note**: A case can only be synced with one external resource at a time, per project. To enable Jira syncing, ServiceNow automatic creation and syncing must be disabled. At this time only cases using the core statuses of "Open", "In Progress" and "Closed" will properly sync with Jira.
```

Comment:
Wasn't sure what "Ongoing" meant in this context, if it's referencing current syncing behavior, recommend removing. If this is a feature name, ignore my suggestion and leave it in!

```suggestion
**Notes**: 
- A case can only be synced with one external resource at a time, per project. To enable Jira syncing, ServiceNow automatic creation and syncing must be disabled. 
- Only cases using the core statuses of "Open", "In Progress" and "Closed" can sync with Jira.
- Two-way syncing requires [webhook support][3].
```